.gitignore

```diff
@@ -11,7 +11,7 @@ ui-dist
 ui/static/font/fontawesome-webfont*
 ui/static/font/KaTeX*
 ui/static/styles.css*
-jest*
+jest_*/
```

Comment:
I'm not sure if this is the right pattern, but on my system there were directories called `jest_0`, so I assumed that was the pattern. Had to change it because otherwise `jest-ts-preprocessor.js` was being ignored, and I wanted that filename to have the word "jest" in it.